### ISSUE TITLE
Create standalone export file

### ISSUE THREAD

**Issue by MidnightLightning** [EdgeApp/edge-login-ui#125]
For projects that don't use a `node_modules` folder or a build process, having a standalone library file would be the ideal solution (possibly submitted to a CDN for simple embedding?).

If some single file library usage option (WebPack?) would be available, that would help those sorts of smaller projects get involved.

**Comment by swansontec** [comment]
I assume this is for the web login UI, not the React Native login UI.

We do export the launcher code as a single-file ES6 module in edge-login-ui-web/lib/edge-login-ui-web.js, but it depends on two outside libraries. It would not be too much work to run this through Webpack / Parcel to make a single-file bundle.

I'll add this to my todo list.

**Comment by MidnightLightning** [comment]
Yes indeed, the web login UI. Having a single, static file would be useful to use as a basic demo, or bootstrapping a site that's not using a build/pack process.
